tests/benchdnn/lnorm/lnorm.cpp

```diff
@@ -640,7 +641,7 @@ std::vector<data_kind_t> get_kinds_to_check(const prb_t *prb) {
 // even if the normalization layer worked correctly
 #if !(DNNL_AARCH64_USE_ACL)
         if (!(prb->flags & GLOB_STATS) && !(prb->dir & FLAG_INF)) {
-            check_kinds.push_back(MEAN);
+            if (!prb->use_rms_norm()) check_kinds.push_back(MEAN);
```

Comment:
Just an anchor comment that decision must be made based on the outcome whether mean remains an output memory or not.

Reply:
resolving as RFC is merged and we agreed on not having mean as an input/output